pkg/controller/selector/selector.go

```diff
@@ -59,7 +48,7 @@ func (s *selector) SelectSupportedPool(tags map[string]interface{}) (*model.Stor
 		return nil, err
 	}
 
-	supportedPools, err := filterChain.Handle(tags, pools)
+	supportedPools, err := SelectSupportedPools(1, tags, pools)
```

Comment:
Can you explain why you select one pool from `SelectSupportedPools`?

Reply:
Consistent with the previous logic, the specific reason can refer to the previous comment of the current function "Now, we just return the firt supported pool which will be improved in the future."